### ISSUE TITLE
issues about vgg54  loss location in network

### ISSUE THREAD

**Issue by yayeoCddy** [tensorlayer/srgan#58]
hi, after reading your codes about the location of vgg54 loss, the location like this
![image](https://user-images.githubusercontent.com/16780698/35723924-f07f847e-0836-11e8-9f80-2802d52fd488.png)

but after reading others'  implementation and read the SRGAN papers for several times, the location like this
![image](https://user-images.githubusercontent.com/16780698/35724075-7ceee846-0837-11e8-8c54-aa4c33471dff.png)

(the code uses the vgg16 network, It doesn't matter. the vgg54 loss locates before last maxpooling layer and is the same as the paper. )
![image](https://user-images.githubusercontent.com/16780698/35724216-ec4b1dc2-0837-11e8-88ae-8fe9cf0cb048.png)

so i feel uncertain.

**Comment by zsdonghao** [comment]
Hi, actually, it doesn't matter, deeper network will have higher level feature extraction. 
and VGG19 is better than VGG16.

**Comment by yayeoCddy** [comment]
yes I know. but I feel uncertain about the location. I mean you put the vgg54loss  after "conv4" before "conv5" in your codes, but someone put it after "vgg_conv5_4" before the last maxpooling(vgg_maxpool5). According to the paper, I think the second location is  right.

**Comment by wagamamaz** [comment]
According to my experiment, it doesn't matter a lot
